scripts/factory-agent.sh

```
@@ -1372,6 +1380,99 @@ ensure_beta_build_root() {
+# declared dead and retried. Comfortably above a cold iOS+Android build.
+FACTORY_LANE_STALE_MIN="${FACTORY_LANE_STALE_MIN:-120}"
+if ! [[ "$FACTORY_LANE_STALE_MIN" =~ ^[1-9][0-9]*$ ]]; then
+  echo "WARNING: invalid FACTORY_LANE_STALE_MIN='$FACTORY_LANE_STALE_MIN'; defaulting to 120" >&2
+  FACTORY_LANE_STALE_MIN=120
+fi
+
+# $1 issue, $2 head sha, $3 pr number.
+intest_check_lane_outcomes() {
+  local issue_num="$1" sha="$2" pr_num="${3:-}"
+  local state_json prior_sha prior_lanes lane exit_file log_file_path code kept="" changed=0
+  local reason attempts_key
```

Shellcheck (0.11.0)

[warning] attempts_key appears unused. Verify use (or export if used externally).

(SC2034)